app/controllers/api_controller/tags.rb

```diff
@@ -94,13 +94,9 @@ def fetch_category(data)
     end
 
     def destroy_tag_and_classification(tag_id)
-      entry = Classification.find_by_tag_id(tag_id)
-      if entry
-        entry.destroy!
-      else
-        tag = Tag.find_by_id(tag_id)
-        tag.destroy! if tag
-      end
+      entry_or_tag = Classification.find_by_tag_id(tag_id) || Tag.find_by_id(tag_id)
+      raise ActiveRecord::RecordNotFound unless entry_or_tag
```

Comment:
I was wondering if there was a reason for using `find_by_id` - I guess that is the reason! Just changed and repushed. Thanks!